plugins/node/opentelemetry-instrumentation-nestjs-core/src/instrumentation.ts

```diff
@@ -22,15 +22,14 @@ import {
   InstrumentationNodeModuleFile,
   isWrapped,
 } from '@opentelemetry/instrumentation';
-import type * as NestJS from '@nestjs/core';
```

Comment:
Aren't next two lines causing the same issue?

Reply:
no, because they are only used in file local functions and not in exported APIs. The .d.ts files hold only the exported types.